src/orion/algo/base.py

```diff
@@ -252,6 +223,7 @@ def observe(self, trials):
     def register(self, trial):
         """Save the trial as one suggested or observed by the algorithm.
 
+        TODO: Update this docstring:
```

Comment:
TODO :)

Reply:
Fixed in 83c4670183e6ca925b2ec00f5024f1a19f601938
(might not be pushed already)